src/coreclr/jit/importer.cpp

```diff
@@ -11528,22 +11528,30 @@ void Compiler::impImportBlock(BasicBlock* block)
         impReimportSpillClique(block);
 
         // For blocks that haven't been imported yet, we still need to mark them as pending import.
-        for (BasicBlock* const succ : block->Succs())
+        // Filter successor from BBJ_EHFILTERRET have already been handled, above.
```

Comment:
I couldn't figure out where the "above" was. Would be nice to avoid this special case.

Seems like we should never get into the spill clique case for filters since the exit stack will be emtpy?

Reply:
Ah, I can remove this from the spill clique case if you think it's unnecessary. (Just assert block is not a filterret?) I was just being defensive, since I hit an assert without this in the "normal" case.